<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FuryFit</class>
 <widget class="QWidget" name="FuryFit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="layoutFuryFit">
   <item>
    <widget class="QGroupBox" name="gbInput">
     <property name="title">
      <string>Input</string>
     </property>
     <layout class="QHBoxLayout" name="furyfit_layoutInput">
      <item>
       <widget class="MantidQt::MantidWidgets::DataSelector" name="dsSampleInput" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="autoLoad" stdset="0">
         <bool>true</bool>
        </property>
        <property name="workspaceSuffixes" stdset="0">
         <stringlist>
          <string>_iqt</string>
         </stringlist>
        </property>
        <property name="fileBrowserSuffixes" stdset="0">
         <stringlist>
          <string>_iqt.nxs</string>
         </stringlist>
        </property>
        <property name="showLoad" stdset="0">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_25">
     <item>
      <layout class="QVBoxLayout" name="furyfit_vlTreeProp">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QLabel" name="furyfit_lbFitType">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Fit Type:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="furyfit_cbFitType">
           <item>
            <property name="text">
             <string>1 Exponential</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2 Exponentials</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1 Stretched Exponential</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1 Stretched Exponential and 1 Exponential</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="1">
          <widget class="QCheckBox" name="furyfit_ckPlotGuess">
           <property name="text">
            <string>Plot Guess</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="furyfit_ckConstrainIntensities">
           <property name="text">
            <string>Constrain Intensities</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="furyfit_ckConstrainBeta">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Constrain Beta Over All Q</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="furyfit_properties"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="furyfit_vlPlot"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="furyfit_gpLegend">
         <item>
          <widget class="QPushButton" name="furyfit_pbSingle">
           <property name="text">
            <string>Fit Single Spectrum</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="furyfit_spLegendSpace">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="furyfit_lbLegendSample">
           <property name="text">
            <string>Sample</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="furyfit_lbLegendFit">
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 0, 0);</string>
           </property>
           <property name="text">
            <string>Fit</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_27">
         <item>
          <widget class="QLabel" name="furyfit_lbSpectra">
           <property name="text">
            <string>Plot Spectrum:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="furyfit_spPlotSpectrum">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_13">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Spectra Range:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="furyfit_spSpectraMin">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>to</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="furyfit_spSpectraMax">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="gbOutput">
     <property name="title">
      <string>Output</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="ckVerbose">
        <property name="text">
         <string>Verbose</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_0">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lbPlotType">
        <property name="text">
         <string>Plot Output: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbPlotType">
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>All</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Intensity</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tau</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Beta</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="ckSave">
        <property name="text">
         <string>Save Result</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MantidQt::MantidWidgets::DataSelector</class>
   <extends>QWidget</extends>
   <header>mantidqt::mantidwidgets::dataselector.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
